packages/flutter/lib/src/material/material.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:ui' as ui show Canvas;
```

Comment:
i'm surprised this isn't already exported by rendering.dart and widgets.dart.

Reply:
done